src/styles/utils/_colors.scss

```diff
@@ -0,0 +1,3 @@
+$days-color:#eee;
```

Comment:
Better use some abstract names for variables
Perhaps you will need to use this color not only for days
You can use something like 
$primary-color
$secondary-color